### DISCUSSION TITLE
client.auth.signInWithPassword works on Android Emulator but not on physical device

### DISCUSSION THREAD

**Discussion by drisme** [supabase/supabase#13617] [General]
I am building an app with Flutter and currently I'm stuck trying to figure out why I can't log in on a physical device. On the Android emulator everything works fine, the moment I try to run it on a mobile device nothing happens, no error redirect, no nothing. I've tried adding the following tags in the AndroidManifest.xml but again nothing.

```
<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
<uses-permission android:name="android.permission.INTERNET" />
```

I found other posts about deep links etc, but why do I suddenly need deep links, I thought I could just use the auth function from the supabase_flutter package?

Any help would be greatly appreciated!

**Comment by dshukertjr** [comment]
Could you share the Dart code you are using?

**Reply by saintonims** [reply]
I have thesame issue, can some one help. I can login and signup with Android emulator but on real devices I can signup or login and no error message pops up

**Reply by dshukertjr** [reply]
@saintonims We wouldn't be able to help you without providing the code or more details of what exactly happens. Would you explain in details what happens, or doesn't happen when you expect something to happen, when you run what code on production Android device?
